openmetadata-service/src/main/java/org/openmetadata/service/resources/glossary/GlossaryTermResource.java

```diff
@@ -244,7 +244,8 @@ public GlossaryTerm get(
       })
   public GlossaryTerm getByName(
       @Context UriInfo uriInfo,
-      @PathParam("name") String name,
+      @Parameter(description = "Name of the glossary term", schema = @Schema(type = "string")) @PathParam("name")
+          String name,
```

Comment:
This should be FullyQualifiedName of the glossary. Also change the parameter name from "name" to "fqn".